registry/config.py

```diff
@@ -38,8 +38,10 @@
 ORG_ID_FOR_TESTING_AI_SERVICE = ""
 BLOCKCHAIN_TEST_ENV = {
     "network_id": 3,
-    "network_name": "ropsten",
+    "network_name": "test",
```

Comment:
the network name should be ropsten, network type will be the "TEST" as mentioned in constants

Reply:
it is dummy config